doc/DeveloperGuide.md

```diff
@@ -88,6 +88,24 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+1. User requests to rename tag
+2. AddressBook shows a list of persons with the tag
+3. AddressBook requests for confirmation
+4. User confirms rename command
+5. AddressBook renames tag for the list of persons
+Use case ends.
+
+**Extensions**
+
+2a. No person has the tags
```

Comment:
This should be `1a.` as it is a scenario that prevents you from going to `2.`
same for the below.